src/germline.cpp

```diff
@@ -15,12 +15,14 @@ namespace linearham {
 /// sites.
 /// @param[in] next_transition
 /// Vector of probabilities of transitioning to the next match state.
+/// @param[in] scaler
+/// An underflow scaler.
 Germline::Germline(Eigen::VectorXd& landing, Eigen::MatrixXd& emission_matrix,
-                   Eigen::VectorXd& next_transition)
-    : emission_matrix_(emission_matrix) {
+                   Eigen::VectorXd& next_transition, double& scaler)
```

Comment:
Surprised me to be passing a reference to a double rather than just the double (we aren't modifying it unless I'm mistaken). Thoughts? @bcclaywell ?